test/integration/scheduler/taint_test.go

```diff
@@ -668,7 +668,7 @@ func TestTaintBasedEvictions(t *testing.T) {
 			testCtx = initTestScheduler(t, testCtx, true, nil)
 			defer cleanupTest(t, testCtx)
 			cs := testCtx.clientSet
-			informers := testCtx.informerFactory
+			informers := informers.NewSharedInformerFactory(testCtx.clientSet, 0)
```

Comment:
We should not share the informer factory between the scheduler and controller. In production, they are different components. 

In volume binding plugin, a pod informer is created on the scheduler informer factory (scheduler itself uses an independent pod informer with custom ListWatch), event handlers registered later by the controller will not take effect.